docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -370,6 +371,16 @@ Indicates whether the container will be auto-updated ([podman-auto-update(1)](po
 
 * `local`: Tells Podman to compare the image a container is using to the image with its raw name in local storage. If an image is updated locally, Podman simply restarts the systemd unit executing the container.
 
+### `CgroupsMode=`
+
+The cgroups mode of the Podman container. Equivalent to the Podman `--cgroups` option.
+
+By default, the cgroups mode of the container created by Quadlet is `split`,
+which differs from the default (`enabled`) used by the Podman CLI.
+
+If the container joins a pod (i.e. `Pod=` is specified), you may change this to
```

Comment:
It's a bit of a nit, but, I would like to try and make this comment better. Reading this made me think that Quadlet will allow you to change it only if the container is part of a pod. So, maybe instead of `you may change` say, `you may want to change`. WDYT?